lib/layer.js

```diff
@@ -173,3 +173,15 @@ function decode_param(val){
     throw err
   }
 }
+
+function once(fn) {
+  var called = false
+
+  return function (err) {
+    if (!err) {
+      if (called) { throw new Error('next() cannot be called twice') }
```

Comment:
I'm referring to the block that is on the line this comment is for, the one with throw in it.